### PR TITLE
Improve cloudfoundry-firehose-nozzle dev docs

### PR DESCRIPTION
**Pull request by pellared**
## Why

Setting up env for testing Cloud Foundry monitoring is not strightforward.

## What

Provide instructions on how to set up an environment for testing the `cloudfoundry-firehose-nozzle` monitor.

### CODE REVIEW

**Review by mstumpfx — APPROVED** [review]

File: pkg/monitors/cloudfoundry/README.md [hunk 1]
```diff
@@ -1,5 +1,102 @@
...
+    uaac client add my-v2-nozzle --name signalfx-nozzle --secret $NOZZLE_SECRET --authorized_grant_types client_credentials,refresh_token --authorities logs.admin
+    ```
+
+### Configure SignalFx Smart Agent monitor
+
+Example config:
+
+```yaml
+---
+signalFxAccessToken: <signafx token>
```
Comment (rmfitzpatrick) [inline pkg/monitors/cloudfoundry/README.md:85]:
```suggestion
signalFxAccessToken: <signalfx token>
```

**Review by rmfitzpatrick — APPROVED** [review]
lgtm though I have 0 experience w/ tanzu.

**Comment by pellared** [comment]
@keitwb @agoallikmaa PTAL

**Comment by agoallikmaa** [comment]
Looks good to me.

**Pull request merged**